### DISCUSSION TITLE
HTML5 Repository for Workzone Standard not loading the React UI5

### DISCUSSION THREAD

**Discussion by rocantonio** [UI5/webcomponents-react#5561] [Q&A]
Hello Team we are deploying SAP UI5 Web Components for React into Workzone Standard but is not loading.

We can open it from HTML5 Applications on Subaccount Level because it uses index.html but from workzone normally it uses component-preload.js any suggestions on how to tackle this error ?

**Comment by CodeWith-Satyajit** [comment]
Any one resolved this issue , I am also facing same issue.
